docs/federation.md

```diff
@@ -0,0 +1,116 @@
+  upgrade the software stack to the latest DACCS stack within two weeks of the updates release date.
+- Node maintenance: Node Administrators accept that joining the Network implies that they are responsible for
+  provisioning, funding and maintaining their own Node.
+- Node security: Node Administrators are responsible for maintaining secure access to their node and for coordinating
+  with
+  the hosting institution regarding any security related considerations prior to the deployment of the Node. Admin
+  access
+  to the Node must be restricted to listed Node Administrators.
+- Downtime policy: A Node should strive to maintain downtime of no more than 5%. If a Node needs to be taken offline for
+  scheduled maintenance purposes, the Node Administrator must provide users at least 2-weeks notice.
+- New user signups: Node Administrators are responsible for the signups of users on their own Nodes. All new signup
+  requests should be verified and approved or rejected within 2 business days.
+- Node configuration: The DACCS deployment stack consists of core components and optional components. A Node can enable
+  or
+  disable non-core components as per the decisions of the Node Administrator. However, when a feature/component is to be
+  removed, adequate notice for the change should be given to the Node’s users.
+
+### Support and Documentation
+
+A Node should maintain a public web-page with:
+
+1. basic information about the Node,
+2. instructions for new users to
+   sign up for an account at the Node,
+3. basic usage policies (as determined by the Node Administrator), and
+4. support information.
+
+We encourage Node Administrators to develop additional documentations and tutorials, if they can,
+as it will be of great value to users on that Node and to other users on the Network, but we do not require it.
+
+Node Administrators are expected to provide technical support to users of their Nodes in a timely manner.
+
+### User Data and Privacy
+
+- Privacy: No Personal Identifying Information belonging to a User can be made accessible to anyone except the User in
+  question, the Node Manager and the DACCS Executive Committee (“Executive Committee”) without explicit written consent
+  of
+  the User in question.
+- No User Generated Data can be made accessible to another User without the User’s permission. without the user’s
+  permission. User Generated Data can not be made accessible outside the Network without the User’s permission.
+- Back-up of User Generated Data: The Node Administrator should have a mechanism in place to regularly back up a user’s
+  main
+  working directory, and have a mechanism in place to retrieve the back-up data upon a user’s request.
+
+### Published Data
+
+Published Data must be discoverable and readable by all Users on a Node. Node Managers are responsible for ensuring that
+all published data is recoverable in case of data loss. Node managers should provide Users at least one-week notice
```

Comment:
Published data I assume we mean Thredds and Geoserver data?  If those are down, most likely it's the network, or electricity or hardware failure.   Will be hard to specify a recovery time-frame.  

Maybe we can say that if we foresee the downtime will be longer than X hours, we will notify the users, instead of leaving them in the dark.

Reply:
Published data is anything that is not editable by a user and is therefore relatively stable. This includes most data served by thredds/geoserver but doesn't include things like wps ouputs, data in user workspaces, etc.

> Maybe we can say that if we foresee the downtime will be longer than X hours, we will notify the users, instead of leaving them in the dark.

I like that idea. I'm happy to prioritize communication with the users over specific recovery policies that may be burdensome to achieve.

Reply:
Why are we saying that published data must be discoverable and readable "by all users on a node" and not "all users on the network?

Reply:
We didn't want to step on a node manager's decision to keep some data private for their own users. Maybe we need to differentiate between:

published data: not editable by users (available to a subset of users in the network) 
public data: available to everyone (no authentication)